07/main07.py

```diff
@@ -0,0 +1,166 @@
+
+phone_dic = {
+    'entry0': None,
+    'entry1': 10,
+    'entry2': 20,
+    'entry3': 30,
+    'entry4': 40,
+    'entry5': 50,
+}
+
+def dic_count_items(c_var_i):
+    counter = 0
+    for key0,value0 in c_var_i.items():
+        counter += 1
+    return counter
+
+
+# decided to do it in cycle, omg here we go
+while True:
+    os.system('cls')
+    inp = input('\n'
+                'Welcome to the test phonebook.\n'
+                'Here is the list of available commands:\n'
+                'stats      - shows you the number of entries stored\n'
+                'add        - adds a new entry\n'
+                'delete     - deletes an entry\n'
+                'list       - shows you all entries in the phonebook\n'
+                'show       - tells you details about entry\n'
+                '\n'
+                'Enter command: ')
+
+    inp_l = str(inp.lower())
+
+    # list ---------------------------
+    if inp_l == 'list':
+        os.system('cls')
+        print('\n')
+        for k, v in phone_dic.items():
+            print(k, v)
+        list_inp = input('\n'
+                        'Enter anything to start over: ')
+        if list_inp is not '':
+            continue
+        else:
+            continue
+    # list ---------------------------
+
+    # stats ---------------------------
+    elif inp_l == 'stats':
+        print(f'\n'
+              f'This phonebook has:\n'
+              f'{dic_count_items(phone_dic)} entries\n'
+              f'')
+    # stats ---------------------------
+
+    # show ---------------------------
+    elif inp_l == 'show':
+        os.system('cls')
+        show_entry = input('\n'
+                          'Type name of the entry to show: ')
+        if show_entry in phone_dic.keys():
+            print(f'\n'
+                  f'Entry "{show_entry} : {phone_dic.get(show_entry)}" is present...')
+            time.sleep(3)
+        else:
+            print(f'\n'
+                  f'Entry "{show_entry}" was not found...')
+            time.sleep(2)
+    # show ---------------------------
+
+    # delete ---------------------------
+    elif inp_l == 'delete':
+        os.system('cls')
+        show_entry = input('\n'
+                          'Type name of the entry to delete: ')
+        if show_entry in phone_dic.keys():
+            del phone_dic[show_entry]
+            print(f'\n'
+                  f'Entry "{show_entry}" with a value has been deleted...')
+            time.sleep(2)
+        else:
+            print(f'\n'
+                  f'Entry {show_entry} was not found...')
+            time.sleep(2)
+    # delete ---------------------------
+
+    # add ---------------------------
+    elif inp_l == 'add':
+        os.system('cls')
+        new_entry = input('\n'
+                          'Type name for the new entry: ')
+        if new_entry not in phone_dic:
+            new_value = input('\n'
+                              'Type name for the new value: ')
```

Comment:
You are asking for a name here, but I guess you should ask for a phone number

Reply:
I agree. I have now fixed that (it should show up in a new commit).